### PR TITLE
Merge 4.5.1-7.16 into 4.6.0-7.16

### PR DESCRIPTION
**Pull request by Machi3mfl**
### Description

Merge 4.5.1-7.16 into 4.6.0-7.16

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,150 @@ You can learn more about it at [wazuh.com][web]
...
+```
+chown -R kibana:kibana /usr/share/kibana/data
+chown -R kibana:kibana /usr/share/kibana/plugins
+```
+
+Install the Wazuh app
+
+```
+cd /usr/share/kibana/
+sudo -u kibana bin/kibana-plugin install https://packages.wazuh.com/4.x/ui/kibana/wazuh_kibana-4.5.1_7.16.0-1.zip
```
Comment (Desvelao) [inline README.md:217]: **nitpick**: this was changed here: https://github.com/wazuh/wazuh-kibana-app/pull/5403/files

File: README.md
Comment (Desvelao) [inline README.md:1]: **nitpick**: the changes related to install/upgrading the plugin should be done in the `plugins/main/README.md` and should have into account the pull request https://github.com/wazuh/wazuh-kibana-app/pull/5403/files.

File: README.md [hunk 2]
```diff
@@ -98,6 +98,150 @@ You can learn more about it at [wazuh.com][web]
...
 </p>
 
+## Branches
+
+- `stable` corresponds to the latest Wazuh app stable version.
+- `master` branch contains the latest code, be aware of possible bugs on this branch.
+
+## Requisites
+
+- Wazuh HIDS 4.5.1
```
Comment (Desvelao) [inline README.md:108]: **nitpick**: change to `4.6.0`

File: README.md [hunk 3]
```diff
@@ -98,6 +98,150 @@ You can learn more about it at [wazuh.com][web]
...
+```
+mkdir /usr/share/kibana/data
+chown -R kibana:kibana /usr/share/kibana/data
+```
+
+Install the Wazuh app plugin for Kibana
+
+```
+cd /usr/share/kibana
+sudo -u kibana bin/kibana-plugin install https://packages.wazuh.com/4.x/ui/kibana/wazuh_kibana-4.5.1_7.16.0-1.zip
```
Comment (Desvelao) [inline README.md:126]: **nitpick**: this should have into account this pull request https://github.com/wazuh/wazuh-kibana-app/pull/5403/files

File: plugins/main/public/components/common/welcome/agents-welcome.js [hunk 4]
```diff
@@ -360,31 +446,37 @@ class AgentsWelcome extends Component {
...
+            <div
+              style={{
+                display:
+                  this.props.resultState === 'loading' ? 'block' : 'none',
+                alignSelf: 'center',
+                paddingTop: 100,
+              }}
+            >
+              <EuiLoadingChart size='xl' />
+            </div>
```
Comment (Desvelao) [inline plugins/main/public/components/common/welcome/agents-welcome.js:478]: **nitpick**: this seems there was a problem solving the conflict. The method should render a button related to MITRE ATT&CK. See the current logic in the target branch: https://github.com/wazuh/wazuh-kibana-app/blob/4.6.0-7.16/plugins/main/public/components/common/welcome/agents-welcome.js#L356-L392

File: plugins/main/public/components/common/welcome/agents-welcome.js [hunk 5]
```diff
@@ -470,84 +562,154 @@ class AgentsWelcome extends Component {
...
+        </EuiFlexGroup>
+      );
+    }
 
-              <div className='wz-module-header-nav'>
-                <div>
-                  <EuiPanel grow paddingSize='s' className="wz-welcome-page-agent-info">
-                    <AgentInfo agent={this.props.agent} isCondensed={false} hideActions={true} {...this.props}></AgentInfo>
-                  </EuiPanel>
+    render() {
```
Comment (Desvelao) [inline plugins/main/public/components/common/welcome/agents-welcome.js:573]: **nitpick**: this method is duplicated.

File: plugins/main/public/components/common/welcome/agents-welcome.js [hunk 6]
```diff
@@ -470,84 +562,154 @@ class AgentsWelcome extends Component {
...
-      <div className="wz-module wz-module-welcome">
-        <div className='wz-module-header-agent wz-module-header-agent-wrapper'>
-          <div className='wz-module-header-agent-main'>
-            {title}
-          </div>
-        </div>
-        <div className='wz-module-agent-body wz-module-agents-padding-responsive'>
-          <EuiPage>
-            <EuiPageBody component="div">
+    renderSCALastScan() {
```
Comment (Desvelao) [inline plugins/main/public/components/common/welcome/agents-welcome.js:565]: **nitpick**: this method is duplicated.

File: plugins/main/scripts/tag.py [hunk 7]
```diff
@@ -17,7 +17,7 @@
 # Wazuh version: major.minor.patch
 version = '4.6.0'
 # App's revision number (previous rev + 1)
-revision = '01'
+revision = '02'
```
Comment (Desvelao) [inline plugins/main/scripts/tag.py:20]: **nitpick**: undone the change

File: CHANGELOG.md [hunk 8]
```diff
@@ -54,14 +54,40 @@ All notable changes to the Wazuh app project will be documented in this file.
...
+- Fixed the agents active coverage stat as NaN in Details panel of Agents section [#5490](https://github.com/wazuh/wazuh-kibana-app/pull/5490)
+
+### Removed
+
+- Removed the agent name in the agent info ribbon [#5497](https://github.com/wazuh/wazuh-kibana-app/pull/5497)
+
+### Changed
+
+- Changed method to perform redirection on agent table buttons [#5539](https://github.com/wazuh/wazuh-kibana-app/pull/5539)
+- Changed windows agent service name in the deploy agent wizard [#5538](https://github.com/wazuh/wazuh-kibana-app/pull/5538)
```
Comment (Desvelao) [inline CHANGELOG.md:75]: **nitpick**: add a new line.

**Comment by Machi3mfl** [comment]
## Evidence


### :red_circle:  Duplicated evolution card

image

### :green_circle:  Agents sections

- Agents ipv6 render

image

image

image

image



Testing: :heavy_check_mark: 
CR: :hourglass: In progress

**Review by Desvelao — DISMISSED** [review]
**review**

:green_circle: code
:green_circle: test

image
image

File: plugins/main/public/components/common/welcome/agents-welcome.js
[the remark this answers is not in the document]
Comment (Desvelao) [reply plugins/main/public/components/common/welcome/agents-welcome.js:1]: Solved https://github.com/wazuh/wazuh-kibana-app/pull/5698/commits/85ab302fd365207e6399c2d5d3cbe02b96856774.

File: plugins/main/public/components/common/welcome/agents-welcome.js [hunk 9]
```diff
@@ -470,84 +562,154 @@ class AgentsWelcome extends Component {
...
+                  manager.
+                </p>
+                <a
+                  href={webDocumentationLink(
+                    'user-manual/agents/agent-connection.html',
+                  )}
+                  target='_blank'
+                >
+                  Checking connection with the Wazuh server
+                </a>
```
[the remark this answers is not in the document]
Comment (Desvelao) [reply plugins/main/public/components/common/welcome/agents-welcome.js]: Solved https://github.com/wazuh/wazuh-kibana-app/pull/5698/commits/85ab302fd365207e6399c2d5d3cbe02b96856774.

File: README.md
Comment (Desvelao) [reply README.md:1]: Solved https://github.com/wazuh/wazuh-kibana-app/pull/5698/commits/85ab302fd365207e6399c2d5d3cbe02b96856774.

File: README.md (hunk 1 above)
Comment (Desvelao) [reply README.md]: Solved https://github.com/wazuh/wazuh-kibana-app/pull/5698/commits/85ab302fd365207e6399c2d5d3cbe02b96856774.

File: README.md (hunk 2 above)
Comment (Desvelao) [reply README.md]: Solved https://github.com/wazuh/wazuh-kibana-app/pull/5698/commits/85ab302fd365207e6399c2d5d3cbe02b96856774.

File: README.md (hunk 3 above)
Comment (Desvelao) [reply README.md]: Solved https://github.com/wazuh/wazuh-kibana-app/pull/5698/commits/85ab302fd365207e6399c2d5d3cbe02b96856774.

File: plugins/main/public/components/common/welcome/agents-welcome.js (hunk 4 above)
Comment (Desvelao) [reply plugins/main/public/components/common/welcome/agents-welcome.js:464]: Solved https://github.com/wazuh/wazuh-kibana-app/pull/5698/commits/85ab302fd365207e6399c2d5d3cbe02b96856774.

File: plugins/main/scripts/tag.py (hunk 7 above)
Comment (Desvelao) [reply plugins/main/scripts/tag.py]: Solved https://github.com/wazuh/wazuh-kibana-app/pull/5698/commits/85ab302fd365207e6399c2d5d3cbe02b96856774.

File: CHANGELOG.md (hunk 8 above)
Comment (Desvelao) [reply CHANGELOG.md:74]: Solved https://github.com/wazuh/wazuh-kibana-app/pull/5698/commits/85ab302fd365207e6399c2d5d3cbe02b96856774.

File: plugins/main/public/components/common/welcome/agents-welcome.js (hunk 6 above)
Comment (Desvelao) [reply plugins/main/public/components/common/welcome/agents-welcome.js:470]: Solved https://github.com/wazuh/wazuh-kibana-app/pull/5698/commits/85ab302fd365207e6399c2d5d3cbe02b96856774.

File: plugins/main/public/components/common/welcome/agents-welcome.js (hunk 5 above)
Comment (Desvelao) [reply plugins/main/public/components/common/welcome/agents-welcome.js:478]: Solved https://github.com/wazuh/wazuh-kibana-app/pull/5698/commits/85ab302fd365207e6399c2d5d3cbe02b96856774.

**Review by yenienserrano — APPROVED** [review]
LGTM!

**Pull request merged**